mod_reforged/hooks/entity/world/locations/legendary/black_monolith_location.nut

```diff
@@ -1,4 +1,8 @@
 ::Reforged.HooksMod.hook("scripts/entity/world/locations/legendary/black_monolith_location", function(q) {
+	// These fields are used to spawn new entities before combat
+	q.m.NumTroopsMax <- 47;
```

Comment:
My idea with that member variable was so that submodders can tweak that value for upper limit of total troops during troop recovery before every battle. If we set it dynamically during `onSpawned` then modders have to hook `onSpawned` to set that value. Maybe that is fine. What do you think about this?

Reply:
Ok, hard-coded sounds good enough until we know more about what we wanna do with this location